tools/benchmark/src/resultFormatting.ts

```diff
@@ -86,9 +87,9 @@ export function addCells(
 			cell.cell(table, data);
 		}
 	}
-
-	// Add extra cells
-	for (const key of keys) {
-		table.cell(key, data[key]);
+	// Add custom data cells
+	for (const [key, value] of Object.entries(data)) {
+		const displayValue = dataFormatter[key](value);
```

Comment:
The data formatter for a given key could be undefined, we need a check for that and if so we just print `value`